src/main/ank/1.1-category-the-essence-of-composition.md

```diff
@@ -0,0 +1,60 @@
+```Haskell
+f :: A -> B
+```
+```kotlin
+val f: (A) -> B
+```
+................
+```Haskell
+g :: B -> C
+```
+```kotlin
+val g: (B) -> C
+```
+...............
+```Haskell
+g . f
+```
+```kotlin
+g compose f
+```
+..............
+```Haskell
+f :: A -> B
+g :: B -> C
+h :: C -> D
+h . (g . f) == (h . g) . f == h . g . f
+```
+```kotlin
+val f: (A) -> B
+val g: (B) -> C
+val h: (C) -> D
+
+h compose (g compose f) === (h compose g) compose f === h compose g compose f
+```
+..............
+```Haskell
+id :: a -> a
+id x = x
+```
+```kotlin:ank:silent
+fun <A> identity(a: A): A = a
+```
+..............
+```Haskell
+f . id == f
+id . f == f
+```
+```kotlin:ank:replace
+import arrow.core.compose
+import arrow.core.identity
+
+val f: (Int) -> Int = { it }
+val identity: (Int) -> Int = ::identity
+""
```

Comment:
Typo?

Reply:
Nope, just an abuse of `ank:replace` that replaces the block with the last return statement, in this case `""`